articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -359,14 +359,49 @@ In individual trials, you directly control the model architecture and hyperparam
 Object detection | **YOLOv5**: One stage object detection model   <br>  **Faster RCNN ResNet FPN**: Two stage object detection models  <br> **RetinaNet ResNet FPN**: address class imbalance with Focal Loss <br> <br>*Note: Refer to [`model_size` hyperparameter](reference-automl-images-hyperparameters.md#model-specific-hyperparameters) for YOLOv5 model sizes.*| ***`yolov5`\**** <br> `fasterrcnn_resnet18_fpn` <br> `fasterrcnn_resnet34_fpn` <br> `fasterrcnn_resnet50_fpn` <br> `fasterrcnn_resnet101_fpn` <br> `fasterrcnn_resnet152_fpn` <br> `retinanet_resnet50_fpn` 
 Instance segmentation | **MaskRCNN ResNet FPN**| `maskrcnn_resnet18_fpn` <br> `maskrcnn_resnet34_fpn` <br> ***`maskrcnn_resnet50_fpn`\****  <br> `maskrcnn_resnet101_fpn` <br> `maskrcnn_resnet152_fpn`
 
+#### Supported model architectures - HuggingFace and MMDetection (preview)
+
+With the new backend that runs on [AzureML Pipelines](concept-ml-pipelines.md), you can additionally use any image classification model from the [HuggingFace Hub](https://huggingface.co/models?pipeline_tag=image-classification&library=transformers) which is part of the transformers library (such as microsoft/beit-base-patch16-224), as well as any object detection or instance segmentation model from the [MMDetection Version 2.28.2 Model Zoo](https://mmdetection.readthedocs.io/en/v2.28.2/model_zoo.html) (such as atss_r50_fpn_1x_coco). 
+
+In addition to supporting any model from HuggingFace Transfomers and MMDetection 2.28.2, we also offer a list of curated models from these libraries in the azureml-staging registry. These curated models have been tested thoroughly and use default hyperparameters selected from extensive benchmarking to ensure effective training. The table below summarizes these curated models.
+
+Task |  model architectures | String literal syntax
+---|----------|----------
+Image classification<br> (multi-class and multi-label)| **BEiT**: A Vision Transformer (ViT), which is a transformer encoder model (BERT-like). |[`microsoft/beit-base-patch16-224-pt22k-ft22k`](https://ml.azure.com/registries/azureml-preview/models/microsoft-beit-base-patch16-224-pt22k-ft22k/version/1)<br> [`google/vit-base-patch16-224`](https://ml.azure.com/registries/azureml-preview/models/google-vit-base-patch16-224/version/1)<br> [`facebook/deit-base-patch16-224`](https://ml.azure.com/registries/azureml-preview/models/facebook-deit-base-patch16-224/version/1)<br> [`microsoft/swinv2-base-patch4-window12-192-22k`](https://ml.azure.com/registries/azureml-preview/models/microsoft-swinv2-base-patch4-window12-192-22k/version/1)
```

Comment:
![image](https://github.com/MicrosoftDocs/azure-docs/assets/3809388/75d62815-3508-4035-a3ae-acd226a56671)


missing the description for other models apart from Beit - we should have one line for each model family like for the legacy ones; also, not sure we should put the links, these are not very useful for automl users